training/cogvideox_image_to_video_lora.py

```diff
@@ -308,6 +371,11 @@ def main(args):
         variant=args.variant,
     )
 
+    if args.ignore_learned_positional_embeddings:
+        del transformer.patch_embed.pos_embedding
+        transformer.patch_embed.use_learned_positional_embeddings = False
+        transformer.config.use_learned_positional_embeddings = False
```

Comment:
Smooth!

Reply:
Worth noting that these changes would also be required when running inference with a model trained this way because Diffusers will error-out otherwise

Reply:
That makes sense. Let's make a note about it then!